.gitignore

```diff
@@ -44,5 +44,5 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
-yarn.lock
+# yarn.lock
 package-lock.json
```

Comment:
I can't understand why this is the way it is and yarn.lock is still being commited.